src/main.rs

```diff
@@ -1,3 +1,92 @@
+extern crate clap;
+
+use clap::{Arg, App, SubCommand};
+
 fn main() {
-    println!("Hello, world!");
+    let gpio_cmd_matches = App::new("GPIO Utils")
+        .version(env!("CARGO_PKG_VERSION"))
+        .about("Read, Write, and Configure GPIOs")
+
+        // Global options
+        .arg(Arg::with_name("config")
```

Comment:
Have you given any thought to what to do with conflicting pin declarations, since there could possibly be multiple config files?